### ISSUE TITLE
Regression in default user-agent caused by #713

### ISSUE THREAD

**Issue by nehaljwani** [adrienverge/openfortivpn#745]
If openfortivpn is spawned without a config file after [this commit](https://github.com/adrienverge/openfortivpn/pull/713/commits/ba7622feee2568e6c6188d8ee42fd95bb288620b), VPN tunnel fails to establish because of blank user-agent.

See the debug output below:
![Screenshot from 2020-07-11 19-54-14](https://user-images.githubusercontent.com/1779189/87235942-4f47fe00-c3b0-11ea-84c1-f465f134634d.png)

**Comment by nehaljwani** [comment]
I spawn openfortivpn via CLI like this:

```openfortivpn --use-resolvconf=0 -vvv vpn.mycompany.com 443 --username=nwani --otp=1```

**Comment by DimitriPapadopoulos** [comment]
I cannot reproduce this. The user agent is always **`User agent: Mozilla/5.0 SV1`** for me. How to reproduce this?

**Comment by nehaljwani** [comment]
I am always able to reproduce this after https://github.com/adrienverge/openfortivpn/pull/713/commits/ba7622feee2568e6c6188d8ee42fd95bb288620b

This is where the default was removed: https://github.com/adrienverge/openfortivpn/pull/713/commits/ba7622feee2568e6c6188d8ee42fd95bb288620b#diff-702344bade0f7e492b07be2d38ce5110L277-R279

I spawn it like this:
```
/path/to/compiled/openfortivpn --use-resolvconf=0 -vvv vpn.mycompany.com 443 --username=<username> --otp=1
```

**Comment by DimitriPapadopoulos** [comment]
Ah, I can reproduce it now. I had removed the wrong config file while testing.

**Comment by DimitriPapadopoulos** [comment]
I have an alternate fix: #752. It removes code instead of adding code. I don't think _strdup()_ is required here. Would it be OK with you?

**Comment by nehaljwani** [comment]
Yes, seems good to me!
